service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/base/AbstractEmailContentProvider.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.hmcts.reform.fpl.service.email.content.base;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.AccessLevel;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor(access = AccessLevel.PROTECTED)
+public abstract class AbstractEmailContentProvider {
```

Comment:
I know that you just merged it from mine but thoughts on having 2 constructors in here, in other words a new one which overloads the current one with only a single parameter.
```java
    public AbstractEmailContentProvider(String uiBaseUrl) {
        this(uiBaseUrl, null);
    }
```
and then the constructors in `UpcomingHearingsContentProvider` and `FailedPBAPaymentContentProvider` don't need to pass `null` themselves which may look wrong